pkg/reduce/pbq/store/wal/segment_test.go

```diff
@@ -261,7 +261,7 @@ func Test_batchSyncWithMaxBatchSize(t *testing.T) {
 	assert.NoError(t, err)
 	err = wal.Write(&message)
 	assert.NoError(t, err)
-	assert.Equal(t, int64(222), tempWAL.prevSyncedWOffset)
+	assert.Equal(t, int64(252), tempWAL.prevSyncedWOffset)
```

Comment:
because the ID changed to include vertex name, partition, etc, it made the overall message longer.